WalletWasabi/TorSocks5/TorProcessManager.cs

```diff
@@ -68,47 +68,47 @@ public void Start(bool ensureRunning, string dataDir)
 						}
 
 						var fullBaseDirectory = Path.GetFullPath(AppContext.BaseDirectory);
-						if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-						{
-							if (!fullBaseDirectory.StartsWith('/'))
-							{
-								fullBaseDirectory.Insert(0, "/");
-							}
-						}
```

Comment:
Removing the immediate correction of `fullBaseDirectory` after the declaration of it makes the readability worse.

Reply:
We can declare `fullBaseDirectory` after we declare `torPath`.
I moved that part of the code just to not check `if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))` two times.